shell_integration/windows/CMakeLists.txt

```diff
@@ -7,9 +7,16 @@ include_directories(
     ${CMAKE_CURRENT_BINARY_DIR}
 )
 configure_file(WinShellExtConstants.h.in ${CMAKE_CURRENT_BINARY_DIR}/WinShellExtConstants.h)
-configure_file(WinShellExt.wxs.in ${CMAKE_CURRENT_BINARY_DIR}/WinShellExt.wxs)
 
 add_subdirectory(NCContextMenu)
 add_subdirectory(NCOverlays)
 add_subdirectory(NCUtil)
 
+if(BUILD_WIN_MSI)
+    configure_file(WinShellExt.wxs.in ${CMAKE_CURRENT_BINARY_DIR}/WinShellExt.wxs)
+
+    install(FILES
```

Comment:
This is like the other installs in the project too ^^

1. Build into intermediate dir
2. Install if desired

Or not? ^^

Reply:
Well the intent is that I run "make" all the build artifact end up in the right place and it builds the stuff while "make install" puts what was built in the install folder(s).

There if I read it correctly a make install would be necessary for the msi generation to work, which feels odd to me.

Reply:
Guess this just feels counter-intuitive because it's Windows ;-)

The actual "install" of course doesn't make sense at first, like it would do on Linux. The install step puts all release artifacts in the right place, which would otherwise just end up in the the build folders. For example, "nextcloud.exe" and a few DLLs and other files go to the "bin" folder. So I simply hooked into this procedure to put a "msi" folder at the same install root level. You best see it when you the `client-building/build.bat Release` script once and examine the "install" folder.